hawkauthlib/__init__.py

```diff
@@ -52,7 +52,7 @@
 
 
 @utils.normalize_request_object
-def sign_request(request, id, key, algorithm=None, params=None):
+def sign_request(request, _id, key, algorithm=None, params=None):
```

Comment:
Is this to avoid shadowing the building `id` name?  If so, what do you think about `id_` in the style suggested by PEP8 rather than `_id` which looks like a private variable?

Reply:
right `id` is a python keyword and `id_` is much better / I will fix it.